backends-clickhouse/src/main/java/io/glutenproject/memory/alloc/CHNativeMemoryAllocators.java

```diff
@@ -57,6 +60,7 @@ public static CHNativeMemoryAllocator contextInstance() {
     if (!TaskResources.isResourceRegistered(id)) {
       final CHNativeMemoryAllocatorManager manager =
           createNativeMemoryAllocatorManager(
+              "contextInstance",
```

Comment:
Why the name starts with lower case? I see it's upper case from other usages.

Reply:
Already update.